### ISSUE TITLE
upgrading from polling to websocket to establish a connection with the backend, causing the problem of duplicate sessionId

### ISSUE THREAD

**Issue by cunyouchen** [mrniko/netty-socketio#865]
If the front end js first initiates polling requests, and then upgrades to websocket, the polling request will first generate a UUID as the sessionId, which will be added to the io field of the cookie. No matter how many connections are established later, the sessionId will be the cookie The value of io, which will cause the new connection to knock off the old one. The same browser cannot have multiple connections at the same time. I don't know whether the framework is designed in this way and has such a usage scenario, or is it a bug?
Code location：
1.AuthorizeHandler.generateOrGetSessionIdFromRequest(HttpHeaders headers); （Line 245）
2.EncoderHandler.sendMessage();  （Line 141）

**Comment by cunyouchen** [comment]
yes, we solved it in the same way. But I am more interested in the design, or a bug?

**Comment by Toifyx** [comment]
> yes, we solved it in the same way. But I am more interested in the design, or a bug?

sorry, i hava no idea
